app/main/views/g_cloud.py

```diff
@@ -29,7 +30,10 @@
 
 @main.route('/g-cloud')
 def index_g_cloud():
-    return render_template('index-g-cloud.html')
+    show_search_box = not feature.is_active('GCLOUD9')
+
+    return render_template('index-g-cloud.html',
```

Comment:
All the lots and everything for G-Cloud 9 are wrong in that template.  Rather than building up a huge sequence of conditionals in that file, would it be better to have the feature flag switch to a new `index-g-cloud` template altogether (with a different name, but one that doesn't specifically include a `9` in it), with no search box and the right lots?

And then after G9 goes live we can remove the old template along with the feature flag.